app/views/install/compose.phtml

```diff
@@ -153,6 +156,51 @@ services:
       - _APP_DB_PASS
       - _APP_USAGE_STATS
 
+  appwrite-executor:
+    image: <?php echo $organization; ?>/<?php echo $image; ?>:<?php echo $version."\n"; ?>
+    entrypoint: executor
+    container_name: appwrite-executor
+    stop_signal: SIGINT
+    networks:
+      appwrite:
+      runtimes:
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - appwrite-functions:/storage/functions:rw
+      - /tmp:/tmp:rw
+    depends_on:
+      - redis
+      - mariadb
+      - appwrite
+    environment:
+      - _APP_ENV
+      - _APP_OPENSSL_KEY_V1
+      - _APP_REDIS_HOST
+      - _APP_REDIS_PORT
+      - _APP_REDIS_USER
+      - _APP_REDIS_PASS
+      - _APP_DB_HOST
+      - _APP_DB_PORT
+      - _APP_DB_SCHEMA
+      - _APP_DB_USER
+      - _APP_DB_PASS
```

Comment:
This block is not synced with the main compose file and uses a lot of un-required env vars.